src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -60,6 +61,8 @@ def __init__(
             Normal,
         ],
         initrv: typing.Optional[Normal] = None,
+        diffusion=None,
```

Comment:
the diffusion class is passed as a class. Strings are only communicated inside `probsolve_ivp`. I suppose we will add more diffusion soon, and bookkeeping the strings is a bit tedious...